package.json

```diff
@@ -80,7 +81,7 @@
     "collectCoverage": true,
     "coverageThreshold": {
       "global": {
-        "branches": 94,
+        "branches": 93,
```

Comment:
😢

Reply:
I missed to test [this branch](https://github.com/saucelabs/node-saucelabs/blob/cb-sc/src/index.js#L227):

```js
.use(process.platform.startsWith('win') ? 'sc.exe' : 'sc')
```

which seems very obvious and I don't think that we touch any of this code.